ui/narrative/methods/build_genome_set_from_tree/display.yaml

```diff
@@ -54,6 +54,8 @@ parameters :
 
 description : |
     <p>This method allows a user to extract a set of genomes from a user-generated species Tree object. First, this method copies the set of genomes from the tree to the current narrative.  The output widget allows to name the Genome Set object as well as to select specific genomes from the genomes contained in the input Tree object. The genomes can be deleted from the output set by clicking the trash can icon.  Additional genomes can be added by pressing &quot;Add&quot; button at the bottom of the table in the Narrative panel. Finally, the Genome Set object is saved when the &quot;Save&quot; button is clicked. The resulting Genome Set object can then be submitted to additional methods such as <a data-method-id=Òcompute_pangenomeÓ>Compute Pangenome</a> and <a data-method-id=Òinsert_genomeset_into_species_tree_genericÓ>Insert Set of Genomes into Species Tree</a>.</p>
-    
-    <p><strong>Team members who developed & deployed algorithm in KBase:</strong> Roman Sutormin. For questions, <a href="mailto:help@kbase.us">e-mail help@kbase.us</a></p>
+   
+<p>Note: when inserting one or more genomes into a species tree, the inserted genomes that are also contained within the KBase list of species will be duplicated in the tree. One copy will have the KBase list ID and the other will have the ID of the inserted genome. The genomes will also be duplicated in any genome set generated from the tree.</p>
```

Comment:
I think you should keep standard indentation (the same number of spaces on left side of line) as for other lines in description block.